packages/react-data-grid-react-window/src/components/DataGridBody/renderDataGridBody.tsx

```diff
@@ -0,0 +1,31 @@
+/** @jsxRuntime classic */
+/** @jsx createElement */
+
+import { createElement } from '@fluentui/react-jsx-runtime';
+import { getSlots } from '@fluentui/react-components';
+import type {
+  DataGridBodyState,
+  DataGridBodySlots,
+} from './DataGridBody.types';
+import { FixedSizeList as List } from 'react-window';
+
+/**
+ * Render the final JSX of DataGridVirtualizedBody
+ */
+export const renderDataGridBody_unstable = (state: DataGridBodyState) => {
+  const { slots, slotProps } = getSlots<DataGridBodySlots>(state);
```

Comment:
This originally used `getSlotsNext` is there a reason this isn't a part of the react-components public API? what's the plan going forward?